src/views/Home.jsx

```diff
@@ -0,0 +1,9 @@
+import Cover from '../components/Cover'
+
+const Home = () => {
+  return(
+    <Cover />
+  )
+}
```

Comment:
Dado que es un return de un solo componente, sugiero dejarlo así:
```suggestion
const Home = () => {
  return <Cover />;
}
```